source/deploy/mobile-app-changelog.md

```diff
@@ -1,12 +1,38 @@
 # Mattermost Mobile Apps Changelog
 
 Latest Mattermost Mobile Apps releases:
-- [1.49.1 Release](#release)
-- [1.49.0 Release](#id1)
-- [1.48.2 Release](#id4)
-- [1.48.1 Release](#id9)
-- [1.48.0 Release](#id14)
-- [1.47.2 Release](#id20)
+- [1.50.0 Release](#release)
+- [1.49.1 Release](#id1)
+- [1.49.0 Release](#id4)
+- [1.48.2 Release](#id10)
+- [1.48.1 Release](#id15)
+- [1.48.0 Release](#id20)
+
+## 1.50.0 Release
+- Release Date: March 16, 2022
+- Server Versions Supported: Server v5.37.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v5.37.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.31 has ended and upgrading to server ESR v5.37.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Added support for markdown inline image custom sizes.
+ - Added support for showing the release status when uploading a build to Google Play store.
```

Comment:
Are this and the next one public facing changes?

Reply:
@enahum I think you reviewed those PRs, let me know what you think.